Sources/MapboxMaps/Style/StyleErrors.swift

```diff
@@ -26,7 +30,7 @@ public enum SourceError: Error {
 /// Error enum for all layer-related errors
 public enum LayerError: Error {
     /// The layer provided to the map in `addLayer()` could not be encoded
-    case layerEncodingFailed(Error)
+    case layerEncodingFailed(Error?)
```

Comment:
Why is this change needed?

Reply:
Ah, test code left-over. Let's revisit as we look at throwing vs Results. I suspect our error cases will change too.